rand_chacha/Cargo.toml

```diff
@@ -16,6 +16,7 @@ edition = "2021"
 rust-version = "1.61"
 
 [package.metadata.docs.rs]
+features = ["getrandom"]
```

Comment:
I think it's better to follow the CI job and use `all-features = true`. Same with `rand_pcg`.